Trappist/global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "projects": [ "src", "test" ],
   "sdk": {
-    "version": "1.0.0-preview2-1-003177"
+    "version": "1.0.0-preview2-003131"
```

Comment:
If there is an older version in your system then you suppose to update in your system, not in the code